.github/workflows/pipeline.yml

```
@@ -1,95 +1,32 @@
-    # 3. Any workflow without Docker changes: both docker-build and promote-to-latest skipped
-    if: |
-      always() &&
-      ((needs.docker-build.result == 'success' && needs.promote-to-latest.result == 'skipped') ||
-      (needs.promote-to-latest.result == 'success' && needs.docker-build.result == 'skipped') ||
-      (needs.docker-build.result == 'skipped' && needs.promote-to-latest.result == 'skipped'))
-
     runs-on: ubuntu-24.04
 
     container:
-      image: fcsproj/fastcalosim-alma9:${{ needs.docker-check.outputs.container_tag }}
+      image: fcsproj/fastcalosim-alma9:${{ env.container_tag }}
```

actionlint (1.7.4)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)